jekyll/_cci2/circleci-images.md

```diff
@@ -415,6 +387,8 @@ that a given tag has the same meaning across images!
 {% endfor %}
 </ul>
 
+
+<br/>
```

Comment:
My local build needed a bit of space between the generated lists of images and the below `Note:` so I added a BR.